src/encoded/schemas/biosample.json

```diff
@@ -137,9 +137,12 @@
         }
     },
     "facets": {
-        "biosource_summary": {
+        "biosample_type": {
             "title": "Sample Type"
         },
+        "biosample_summary": {
+            "title": "Biosample Summary"
+        },
```

Comment:
I think you meant "biosource_summary" - I didn't see a "biosample_summary" property.